### ISSUE TITLE
internal device IDs vs zone IDs 

### ISSUE THREAD

**Issue by btuysser** [gurumitts/pylutron-caseta#1]
Hi, 

I have a smartbridge PRO and tried your module. I noticed that all the device IDs were off by one compared to the integration report from the lutron app. I believe that commands actually need to act on zone ids rather than device ids. Here is one of the devices : 

            {
                "href": "/device/3",
                "Name": "Cans Livingroom",
                "Parent": {
                    "href": "/project"
                },
                "SerialNumber": 26669871,
                "ModelNumber": "PD-6WCL-XX",
                "DeviceType": "WallDimmer",
                "LocalZones": [
                    {
                        "href": "/zone/2"
                    }
                ]
            }

I believe the command to e.g. turn it off should be : 

#OUTPUT,2,1,0

rather than 

#OUTPUT,3,1,0

Thanks,
   Bijan

**Comment by gurumitts** [comment]
I've come to the same conclusion.  I only have one device so I would have been guessing.  Thanks for validating the assumption.  I'll send out a fix soon.

**Comment by gurumitts** [comment]
So this is what I am seeing on my side:  

From the SSH call: 
```json
{
  "href":"/device/2",
  "Name":"Mud Room",
  "Parent":{"href":"/project"},
  "SerialNumber":26531220,
  "ModelNumber":"PD-6WCL-XX",
  "DeviceType":"WallDimmer",
  "LocalZones":[{"href":"/zone/1"}]
}
```

From the Lutron Integration Report:
```json
  "Zones" : [ {
      "Name" : "Mud Room",
      "ID" : 2
    } ]
```
So my zone id from the ssh call does not match the id I need to send to telnet.  
I will need to investigate further.

**Comment by gurumitts** [comment]
oops did not mean to close the issue.  its reopened.

**Comment by gurumitts** [comment]
this is fixed in 0.2.6
